features/support/pages/pending_projects_page.rb

```diff
@@ -15,7 +15,7 @@ def approve(project:, reason: "I just like it, OK?")
     project_status_change_form.submit(reason: reason)
   end
 
-  def reject(project:, reason: "I don't like it")
+  def reject(project:, reason: "I dislike it")
```

Comment:
APOSTROPHES ARE TURNED INTO `&#rsquo;` AND IT WAS CAUSING MY TEST TO FAIL AND I DIDN'T KNOW WHY